azkaban-common/src/main/java/azkaban/executor/RunningExecutionsUpdater.java

```diff
@@ -40,8 +40,8 @@
   // times (3600 seconds = 1 hour) before we send an email about unresponsive executor.
   private final int numErrorsBetweenUnresponsiveEmail = 360;
   // First email is sent after 1 minute of unresponsiveness
-  private final int numErrorsBeforeUnresponsiveEmail = 6;
-  private final long errorThreshold = 10000;
+  final int numErrorsBeforeUnresponsiveEmail = 6;
```

Comment:
Is there a reason why we wipe out private declaration?

Reply:
This field is read in the unit test to know how many times to run in order to simulate a failure email. Other option would be to hardcode 6 in the test, but I prefer this.